WordPress/src/main/res/layout/delete_site_preference.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ah, good catch.

For one of them, it is suggesting we set a margin to `0dp`. Thankfully we are already doing that, but it's not understanding that `@dimen/margin-none` is `0dp`. Just a lint error.

For all the `Start` and `End` warnings, that is something we have to look at for the app as a whole. Simply adding them for every `Left` and `Right` is not the way to go. I believe this work belongs in #2278

I did however add `android:baselineAligned="false"`, so that hopefully gave us a good bump!